static-site/src/components/nav-bar/index.jsx

```diff
@@ -121,15 +122,19 @@ class NavBar extends React.Component {
         {this.getLogo()}
         {this.getLogoType()}
```

Comment:
I'd suggest making the top-left corner logo/name obviously different than nextstrain.org, e.g. even with something simple like:

```diff
   getLogoType() {
     const title = "Nextstrain";
     const rainbowTitle = title.split("").map((letter, i) =>
       <NavLogoCharacter key={i} colorIndex={i}>{letter}</NavLogoCharacter>
     );
+    const SubTitle = styled.div`
+      color: black;
+    `;
     return (
       this.props.minified ?
         <div/>
         :
         <Link to="/">
           {rainbowTitle}
+          {groupsOnlyService &&
+            <SubTitle>Groups Server</SubTitle>}
         </Link>
     );
   }
```

Reply:
Added your suggestion in [force-push](https://github.com/nextstrain/nextstrain.org/compare/050d4101353bd2af5aea20ccc55f112cbb2bd85c..cfbba8fafc953fe8b1cd7732e887f35b41966685) along with notes to look into making the logo on both static-site and auspice-client customizable.